src/components/dialogs/advanced-search.vue

```diff
@@ -0,0 +1,263 @@
+        <v-icon>mdi-close</v-icon>
+      </v-btn>
+
+      <v-tabs id="advanced-search-tabs">
+        <v-tabs-items v-model="advSearchTab">
+
+          <!-- Advanced Search Form -->
+          <v-tab-item>
+            <v-card-title>
+              <span>Find an Existing Name Request</span>
+            </v-card-title>
+
+            <v-card-text class="copy-normal py-4">
+              Find an existing Name Request by searching for one or more of the following criteria:
+            </v-card-text>
+
+            <!-- Validation error message -->
+            <div v-if="invalidSearch" class="error-message copy-small my-n2">
+              Enter at least one search criteria
+            </div>
+
+            <AdvancedSearchForm
+              class="mt-4"
+              :promptSubmit="toggleSearchSubmit"
+              @isInvalid="invalidSearch = $event"
+              @submitForm="submitForm($event)"
+              @closeDialog="emitClose()"
+            />
+          </v-tab-item>
+
+          <!-- Advanced Search Table -->
+          <v-tab-item>
+            <v-card-title>
+              <span>Name Requests Found (18)</span>
+            </v-card-title>
+
+            <v-card-text class="copy-normal pt-5">
+              Select the Name Request you would like to retrieve. To access the detailed summary screen, <br> you will
+              need the email address of phone number of the Applicant.
+            </v-card-text>
+
+            <AdvancedSearchTable
+              class="mt-6"
+              @selectNameRequest="advSearchTab = AdvancedSearchTabs.ADVANCED_SEACH_RETRIEVE"
+            />
+          </v-tab-item>
+
+          <!-- Advanced Search Retrieve Form -->
+          <v-tab-item>
+            <v-card-title>
+              <span>Retrieve Name Request</span>
+            </v-card-title>
+
+            <v-card-text class="copy-normal pt-5">
+              To view the details and status of this Name Request, please enter the <strong>email address or phone
+              number</strong> of the Applicant.
+            </v-card-text>
+
+            <AdvancedSearchRetrieve
+              class="mt-6"
+              :promptSubmit="toggleRetrieveSubmit"
+              @closeDialog="emitClose()"
+            />
+          </v-tab-item>
+
+          <!-- Invalid Results -->
+          <v-tab-item>
+            <v-card-text class="copy-normal results-text">
+              <h3>Too Many Name Requests Found</h3>
+              <span class="my-6">Your search returned more than 1000 results.</span>
+              <span class="mb-4">Please return to the previous screen and narrow your search by selecting more than
+                  one search criteria and/or reducing your date range.</span>
+            </v-card-text>
+          </v-tab-item>
+
+        </v-tabs-items>
+      </v-tabs>
+
+      <v-card-actions id="advanced-search-actions" class="justify-center pa-0">
+          <v-btn
+            id="adv-search-btn-1"
```

Comment:
This button's vertical alignment is off just a bit.

![image](https://user-images.githubusercontent.com/10002889/117895622-9b372e00-b273-11eb-9f31-fa616be4786f.png)

Reply:
Hmm i'm having a hard time picking it up. Do you mean it's sitting a little high?

Reply:
hmm... yea not sure here. I can see the alignment is centred.. but with no padding influencing the vertical height that i can tell. Didn't do anything unusual here.

Reply:
Yea, the more i look, the icon font size is affecting the alignment from what i can tell. Still looking!

Reply:
It looks like the > is centered vertically but Find Name Request is a pixel or 2 high, compared to Cancel. You may need margin on it to fix it.

Reply:
Yea, the icon itself is increasing that inner elements height a bit. I just shrunk the icon a little, looks better 👍